### PR TITLE
News Item Page

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

**Comment by asasmith** [comment]
There's a sharp line at the edge of the container. Might need to revisit your html structure.

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /news-template/
Browser:   Chrome 91.0.4472.106 on Mac OS 10.15.7
Viewport:  1680 x 914 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-44--airtec.netlify.app%2Fnews-template%2F&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1680x914&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-44--airtec.netlify.app/news-template/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60d12778f84f1d000779b758&commentID=60d33ad932e86e3e9c97cbbf&resolution=resolved)

**Comment by kcbreuil** [comment]
@asasmith that is the container from the carousel, it's showing up because of the drop shadow. See below without shadow: 
Screen Shot 2021-06-23 at 11 56 27 AM

**Comment by asasmith** [comment]
> @asasmith that is the container from the carousel, it's showing up because of the drop shadow. See below without shadow:
> Screen Shot 2021-06-23 at 11 56 27 AM

What about changing the overflow?

File: web/src/components/recentNewsList.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    slidesToScrollPlugin,
+} from '@brainhubeu/react-carousel';
+
+import NewsCard from './newsCard';
+import MoreButton from './moreButton';
+
+const Carousel = loadable(() => import('./carouselWrapper'));
+
+export default function RecentNewsList({ news }) {
+    const text = 'More ❯';
```
Comment (asasmith) [inline web/src/components/recentNewsList.js:16]: `forward` might be a better variable name.

File: web/src/components/newsCard.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ export default function NewsCard({ data }) {
     return (
         <Link
             to="/"
-            className="bg-white h-[34rem] mx-3 md:mr-9 transform transition	focus:scale-110 md:hover:scale-110 overflow-hidden w-80"
+            className="bg-white h-[34rem] mx-3 md:mr-9 transform transition	focus:scale-110 md:hover:scale-110 shadow-xl overflow-hidden w-80"
```
Comment (asasmith) [inline web/src/components/newsCard.js:18]: Remove extra spaces between `transition` and `focus`

**Review by asasmith — CHANGES_REQUESTED** [review]

**Comment by asasmith** [comment]
Still seeing a very sharp and very defined line here

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /news-template/
Browser:   Chrome 91.0.4472.106 on Mac OS 10.15.7
Viewport:  1676 x 914 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-44--airtec.netlify.app%2Fnews-template%2F&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1676x914&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-44--airtec.netlify.app/news-template/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60dc9f5e09cc230008ef8479&commentID=60dca18e46c3a155a0a40a40&resolution=resolved)

**Pull request merged**